website/assemblyscript-react/README.md

```diff
@@ -0,0 +1,112 @@
+<br />
+<br />
+
+<p>
+<img src="https://nearprotocol.com/wp-content/themes/near-19/assets/img/logo.svg?t=1553011311" width="240">
+</p>
+
+<br />
+<br />
+
+## Template for NEAR dapps
+### Requirements
+##### IMPORTANT: Make sure you have the latest version of NEAR Shell and Node Version > 10.x 
```

Comment:
1. It is better to use a bold font (`**text**`) instead of a section.
2. @vgrichina is this statement about Node version > 10.x accurate?

Reply:
I think it's not anymore, we need version > 12.x 

/cc @willemneal

Reply:
yes, everything needs > 12 now. good catch, Frol

Reply:
done